assemblies/karaf-features/src/main/feature/feature.xml

```diff
@@ -211,8 +211,8 @@
     <bundle start-level="82">mvn:org.opencastproject/opencast-incident-workflowoperation/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-index-service/${project.version}</bundle>
     <bundle start-level="82">mvn:org.opencastproject/opencast-ingest-service-api/${project.version}</bundle>
-    <bundle start-level="82">mvn:org.opencastproject/opencast-ingest-service-impl/${project.version}</bundle>
-    <bundle start-level="82">mvn:org.opencastproject/opencast-ingest-workflowoperation/${project.version}</bundle>
+    <bundle start-level="83">mvn:org.opencastproject/opencast-ingest-service-impl/${project.version}</bundle>
```

Comment:
Is there a specific reason to increase the start level?
Usually, that's a sign for broken dependencies

Reply:
I think this was a try to reduce inbox unreliability.

Reply:
I reverted this change